### DISCUSSION TITLE
How to change the name of the binaries produced

### DISCUSSION THREAD

**Discussion by madrigal1** [goreleaser/goreleaser#2038] [Q&A] [answered]
is there any way to change the name of the binaries build . i,e right now its the default of folder name of the application
anyway to customize that?

getting the bleow error when i try to use name-template
https://github.com/goreleaser/goreleaser/issues/2040

**Comment by caarlos0** [comment, marked as answer]
yes, check the docs: https://goreleaser.com/customization/build/

**Reply by siddhuwarrier** [reply]
Just adding a little more context here for the next person who ends up here. If you're building a Golang application, you can specify the build name using the `binary` field in the `builds` section. See https://goreleaser.com/customization/builds/go/

The same appears to apply to the other supported programming languages, but you can see that in the relevant section under https://goreleaser.com/customization/builds
